cmd/smith/app/app.go

```diff
@@ -117,6 +118,10 @@ func (a *App) Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	var catalogger *catalog.Catalog
```

Comment:
I'm really unhappy with the naming for this one, but I'm not sure what it should be.

Reply:
hmm, var catalogClient *catalog.Client is go ish?